apps/settings/locales/settings.en-US.properties

```diff
@@ -843,6 +843,9 @@ fxsync-collections-passwords=Passwords
 fxsync-legal=Legal
 fxsync-tos=Terms of Service
 fxsync-privacy=Privacy Notice
+fxsync-error-invalid-account=This account is inactive
+fxsync-error-invalid-account-explanation=You need to login in Firefox or Firefox for Android and retry
```

Comment:
I think the sentence can be improved a bit, I find the "login in" and "Firefox or Firefox" repetitions a bit hard to read. Maybe:

You need to activate Firefox Sync on Desktop or Android first and then try again.